mcs/class/System.Private.CoreLib/System/Attribute.cs

```diff
@@ -44,8 +44,10 @@ internal static Attribute[] GetCustomAttributes (ICustomAttributeProvider elemen
         {
             if (attributeType == null)
                 throw new ArgumentNullException (nameof (attributeType));
+#if !NETCORE
```

Comment:
Why do you need this if-def?

Reply:
@marek-safar it's just redundant for netcore - it calls `MonoCustomAttrs.GetCustomAttributes` where it checks for `Is Attribute or subclass` again.

Reply:
This code is used only for netcore

Reply:
Oh, indeed